frontend/src/GameReportForm.tsx

```diff
@@ -336,3 +339,43 @@ function GameReportForm() {
 }
 
 export default GameReportForm;
+
+function getStrongholdLabel(stronghold: Stronghold): string {
+    switch (stronghold) {
+        case "Rivendell":
```

Comment:
What is this cursed syntax?? 🤣 Does the input get returned by default if the body of the `case` is empty??

Reply:
Oh, nevermind, I see..

okay.

Reply:
🙃

Reply:
FINALLY I get to use this random language feature for its intended purpose instead of it being a disastrous mistake 😂 it's so cursed